package.json

```diff
@@ -102,6 +102,7 @@
     "lodash.find": "^4.6.0",
     "lodash.includes": "^4.3.0",
     "node-fetch": "^1.6.3",
-    "parse-diff": "^0.4.0"
+    "parse-diff": "^0.4.0",
+    "voca": "^1.2.0"
```

Comment:
Thanks guys for using Voca library! :+1:  (I am the author)

Voca is trying to fill the gap that JavaScript doesn't have an extensive native string library (other than the few methods on the string type itself). The library does not have any dependencies and will never have, because keeping the dependencies at minimum is a good strategy.

A nice bonus is that the library is decomposable into separated functions. If you want to use only the `truncate()` function, you can easily write:
```javascript
import truncate from 'voca/truncate';

truncate('Once upon a time', 7);
```
And no other dependecies are loaded other than related to `truncate()` function.